npm/Fable.Import.Electron.fs

```diff
@@ -516,10 +516,10 @@ module Electron =
         | Normal | Separator | Submenu | Checkbox | Radio
 
     and [<StringEnum>] MenuItemRole =
-        | Undo | Redo | Cut | Copy | Paste | Selectall | Minimize | Close
+        | Undo | Redo | Cut | Copy | Paste | Pasteandmatchstyle | Selectall | Delete | Minimize | Close | Quit | Reload | Forcereload | Toggledevtools | Togglefullscreen | Resetzoom | Zoomin | Zoomout | EditMenu | WindowMenu
```

Comment:
@inosik probably, and it would also be good to put each one on a new line. 

It would be helpful to have a style guide for things like this (although it might not make sense until 1.0)

cc @alfonsogarciacaro